.vscode/settings.json

```diff
@@ -6,4 +6,5 @@
     "storybook-static/static/locales"
   ],
   "jest.jestCommandLine": "yarn test",
+  "i18n-ally.keystyle": "nested",
```

Comment:
We might want to ignore this file from the git repo.

Reply:
Agreed, now that we have the basic settings already tracked in the file for prettier we can untrack it.